src/components/shared/Service/Service.jsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect } from "react";
+import PropTypes from "prop-types";
+
+import { allowScroll, preventScroll } from "../../../utils/scrollUtils";
+import Footer from "../../Footer/Footer";
```

Comment:
일관성을 위해 순서를 변경하면 좋을 것 같아요!

```suggestion
import Footer from "../../Footer/Footer";

import { allowScroll, preventScroll } from "../../../utils/scrollUtils";
```